lib/plugins/aws/lib/naming.js

```diff
@@ -282,8 +282,9 @@ module.exports = {
   getLambdaApiGatewayPermissionLogicalId(functionName) {
     return `${this.getNormalizedFunctionName(functionName)}LambdaPermissionApiGateway`;
   },
-  getLambdaAlexaSkillPermissionLogicalId(functionName) {
-    return `${this.getNormalizedFunctionName(functionName)}LambdaPermissionAlexaSkill`;
+  getLambdaAlexaSkillPermissionLogicalId(functionName, alexaSkillIndex) {
+    return `${this.getNormalizedFunctionName(functionName)}LambdaPermissionAlexaSkill${
```

Comment:
You could default the skill index to 0 here.

Reply:
not sure i'm following you here.. do you want the logical numbering to start from 0 (instead of the current 1 index), or do you want this function to have a default parameter value? (the project supports node v4+, but default parameters were only enabled at node v6).

Reply:
I was just thinking about if it would be allowed to omit the skillIndex here in the function so that it defaults to 0 if it is not set. On the other hand that behavior does not really make sense, because the caller is currently only inside the aws plugin.
Only if a plugin uses this function in `naming` this would matter - but then the call should fail if no index is given if we do not use a default inside.

Reply:
Would you be ok with a solution like
```js
return `${this.getNormalizedFunctionName(functionName)}LambdaPermissionAlexaSkill${
    alexaSkillIndex || 0}`;
```
?

Reply:
Yeah, that sounds reasonable